pkgs/development/compilers/idris2/default.nix

```diff
@@ -30,9 +35,39 @@ stdenv.mkDerivation rec {
 
   checkTarget = "bootstrap-test";
 
-  # idris2 needs to find scheme at runtime to compile
-  postInstall = ''
-    wrapProgram "$out/bin/idris2" --set CHEZ "${chez}/bin/scheme"
+  # TODO: Move this into its down derivation, such that this can be changes
+  #       without having to recompile idris2 every time.
+  postInstall = let
+    packagePaths = builtins.map (l: "$out/${internalVersionString}/" + l) includedLibs;
+    additionalIdris2Paths = builtins.concatStringsSep ":" packagePaths;
+  in ''
+    # Remove existing idris2 wrapper that sets incorrect LD_LIBRARY_PATH
+    rm $out/bin/idris2
+    # Move actual idris2 binary
+    mv $out/bin/idris2_app/idris2.so $out/bin/idris2
+
+    # Remove idris2_app contents
```

Comment:
I can see that this removes `idris2_app` contents, but why?

Reply:
Yes, this comment could be better.

(The actual reason is because it doesn't contain anything that's not contained in `lib` or is not useful for us)